orttraining/orttraining/training_ops/cpu/controlflow/common.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+namespace onnxruntime {
+namespace contrib {
+
+template <int input_start, int output_start>
```

Comment:
This also used in lamb... 
refactor?

Reply:
No. The aliasing logics in Lamb and Event operators can be different.